Add doc for multi value GEO (#3195)
RediSearch/RediSearch: docs/docs/Indexing_JSON.md

  - `null` values are skipped
  - Any other value type is causing an indexing failure

## Index JSON arrays as GEO

> Starting with RediSearch 2.6.1, search can be done on an array of geo (geographical) values or on a JSONPath leading to multiple geo values.

Reviewer: Should we use v2.6?

Author: Some capabilities are available already with 2.6.0 (multi value TEXT)
Some are available with 2.6.1 (multi value NUMERIC and GEO)

Prior to RediSearch 2.6.1, only a single geo value was supported per GEO attribute. The geo value was specified using a comma delimited string in the form "longtitude,latitude", for example, "15.447083,78.238306".

With RediSearch 2.6.1, a JSON array of such geo values is also supported.

> In order to index multiple geo values, either use a JSONPath leading to a single array of geo values, or a JSONPath leading to multiple geo values, using JSONPath operators such as wildcard, filter, union, array slice, and/or recursive descent.

Reviewer 2: ```suggestion
In order to index multiple geo values, use either  a JSONPath leading to a single array of geo values, or a JSONPath leading to multiple geo values, using JSONPath operators such as wildcard, filter, union, array slice, and/or recursive descent.
```

OK

```

And let's add some vendors with their georaphic locations:

```sql
127.0.0.1:6379> JSON.SET vendor:1 $ '{"id":100, "name":"Kwik-E-Mart", "location":["35.213,31.785", "35.178,31.768", "35.827,31.984"]}'
OK

127.0.0.1:6379> JSON.SET vendor:2 $ '{"id":200, "name":"Cypress Creek", "location":["34.638,31.79", "34.639,31.793"]}'
OK

127.0.0.1:6379> JSON.SET vendor:3 $ '{"id":300, "name":"Barneys", "location":["34.648,31.817", "34.638,31.806", "34.65,31.785"]}'
OK
```

To index the `vendor_id` numeric array, specify the JSONPath `$.vendor_id` in the `SCHEMA` definition during index creation:

```sql
127.0.0.1:6379> FT.CREATE itemIdx5 ON JSON PREFIX 1 item: SCHEMA $.vendor_id AS vid NUMERIC
OK
```

To index the `location` geo array, specify the JSONPath `$.location` in the `SCHEMA` definition during index creation:

```sql
127.0.0.1:6379> FT.CREATE vendorIdx ON JSON PREFIX 1 vendor: SCHEMA $.location AS loc GEO
OK
```

> Now we can search for a vendor which is close by to a specific location. For example, a customer is located at geo coordinates 34.5,31.5 and we want to get the vendors which are within the range of 40 km from our location:

Reviewer 2: ```suggestion
Now we can search for a vendor close to a specific location. For example, a customer is located at geo coordinates 34.5,31.5 and we want to get the vendors that are within the range of 40 km from our location:
```
